tests/integration/test_remote_transfer.py

```
@@ -272,16 +263,16 @@ async def test_encryption_headers_sent(
 
     mock_session = MagicMock(wraps=db_session)
     mock_session.close = MagicMock()
-    
+
     with patch("app.services.remote_transfer_service.SessionLocal", return_value=mock_session):
         with patch("app.services.remote_transfer_service.RemoteTransferService._perform_ecdh_key_exchange") as mock_ecdh:
```

Ruff (0.15.9)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

(SIM117)